curator/patterns/reviews.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Title: Reviews
+ * Slug: curator/reviews
+ * Categories: featured, gallery
+ */
+?>
+
+<!-- wp:columns {"verticalAlignment":null,"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%"} -->
+<div class="wp-block-column" style="flex-basis:40%"><!-- wp:image {"id":433,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/sundae.png" alt="" class="wp-image-433"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"10%"} -->
+<div class="wp-block-column" style="flex-basis:10%"></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"center","width":"45%"} -->
+<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:45%"><!-- wp:paragraph {"fontSize":"x-large"} -->
+<p class="has-x-large-font-size"><?php echo esc_html__( 'The Fisherman', 'curator' ); ?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><?php echo esc_html__( 'Nearly all the province of Gallura, washed by the Mediterranean on three sides, consists of mountainous tracts, with valleys intervening, similar to this of the Liscia.', 'curator' ); ?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size"><strong><?php echo esc_html__( '$$$$', 'curator' ); ?></strong><?php echo esc_html__( ' French, European, Healthy', 'curator' ); ?><br><em><?php echo esc_html__( 'Our review: 5/5', 'curator' ); ?></em></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button {"style":{"typography":{"textTransform":"uppercase"}}} -->
+<div class="wp-block-button" style="text-transform:uppercase"><a class="wp-block-button__link wp-element-button"><?php echo esc_html__( 'Get Directions →', 'curator' ); ?></a></div>
```

Comment:
update fontSize to `medium - 18px`, also fontWeight of the arrow alone seems to be set to `400` in designs.